Sming/rboot/eagle.rom.addr.v6.ld

```diff
@@ -0,0 +1,347 @@
+PROVIDE ( Cache_Read_Disable = 0x400047f0 );
```

Comment:
I was using eclipse patch,  but the main point was to know the reference code  for a useful test. 
I let you know.